src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionPlanEndpoint.java

```diff
@@ -69,20 +105,23 @@
   public final ResponseEntity<ActionPlanDTO> createActionPlan(
```

Comment:
So theres complaints from the codebeat about the ABC in random points one being here. Would it be worth splitting it up into two separate methods i.e. one that returns the action plan and one that checks if an action plan name already exists?